app/controllers/two_factor_authentication/webauthn_verification_controller.rb

```diff
@@ -27,7 +27,16 @@ def confirm
       handle_webauthn_result(result)
     end
 
-    def error; end
+    def error
+      flash[:error] = t(
+        'two_factor_authentication.webauthn_error.multiple_methods',
+        link: view_context.link_to(
+          t('two_factor_authentication.webauthn_error.additional_methods_link'),
+          login_two_factor_options_path,
+        ),
+      )
+      redirect_to login_two_factor_webauthn_url(platform: params[:platform])
```

Comment:
Curious, how do the code changes here relate to the ticket. I'm not sure I follow why we'd need them, and the tests pass without them.

Reply:
The original code that was there 
```suggestion
      def error; end
```
is what was redirecting Firefox to the dead end page.
For whatever reason Safari was working without the code change. I may be missing something elsewhere that would make the solution work without this.

Reply:
I think keeping the error view to avoid the 404 with your changes in 8a44fb6826b8f5b13711d982139eee2ea8a53ab6 is enough for this ticket, and we can address the Firefox behavior as part of my work in #8723.

Reply:
I have taken out that change. Thank you.